TripPlanner/Models/Trip.m

```diff
@@ -0,0 +1,103 @@
+//
+//  Trip.m
+//  TripPlanner
+//
+//  Created by Adrienne Li on 7/13/20.
+//  Copyright © 2020 ahli. All rights reserved.
+//
+
+#import "Trip.h"
+
+@implementation Trip
+
+@dynamic tripID;
+@dynamic author;
+@dynamic guests;
+@dynamic images;
+@dynamic description;
+@dynamic title;
+@dynamic location;
+@dynamic startDate;
+@dynamic endDate;
+
++ (nonnull NSString *)parseClassName {
+    return @"Trip";
+}
+
++ (NSString *)dateToString: (NSDate *)createdAt {
+    // format and set createdAtString
+    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+    // configure output format
+    formatter.dateStyle = NSDateFormatterShortStyle;
+    formatter.timeStyle = NSDateFormatterNoStyle;
+    //Convert Date to String
+    NSString *createdAtString = [formatter stringFromDate:createdAt];
+    
+    return createdAtString;
+}
+
++ (void) postUserTrip:(NSMutableArray *)guestUsernames withImages: (NSMutableArray *)images withDescription: (NSString * _Nullable)description withTitle: (NSString *)title withLocation: (NSString *)location withStartDate: (NSDate *)startDate withEndDate: (NSDate *)endDate withGuests: (NSMutableArray *) guests withController:(TripViewController *) controller {
```

Comment:
would it be better to have these functions on a controller instead of the model? Alternatively, instead of a controller you can consider creating a utility class that can do these operations on the given model object.